dotnet/src/SemanticKernel.UnitTests/SkillDefinition/SKFunctionTests3.cs

```diff
@@ -138,13 +138,8 @@ public void Invalid1(string x, string y)
         {
         }
 
-        [SKFunction("two")]
-        public void Invalid2(SKContext cx, string y)
```

Comment:
nice :-)

Reply:
Locally I have a change I'll put up after this to support arbitrary numbers of parameters, automatically pulling from the context variables.